.github/release.yml

```diff
@@ -20,6 +20,9 @@ changelog:
     - title: Bug Fixes
       labels:
         - bug
+    - title: Preview
+      labels:
+        - preview
```

Comment:
Would this be for both the preview rules for the linter and the preview style for the formatter?

Reply:
Yep both! Anything that's not available without the `preview` flag set.